apps/whale-api/src/module.api/poolpair.controller.ts

```diff
@@ -64,7 +64,7 @@ export class PoolPairController {
       return item.id
     })
     response.data = response.data.filter(value => {
-      return !value.symbol.startsWith('BURN-')
+      return !value.symbol.includes('BURN')
```

Comment:
can we add a test for this in https://github.com/JellyfishSDK/jellyfish/blob/main/apps/whale-api/src/module.api/poolpair.controller.e2e.ts